composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/monolog-bundle"               : "~2.7",
         "symfony/swiftmailer-bundle"           : "~2.3",
         "symfony/symfony"                      : "~2.7",
-        "twig/extensions"                      : "~1.2"
+        "twig/extensions"                      : "^1.2"
```

Comment:
I think better to revert it, but I don't think that much necessary, the behavior is the same as [~1.2](https://getcomposer.org/doc/articles/versions.md#caret). I only want to be consistent with other dependencies versions definition via tilde `~`.